src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -31,6 +32,24 @@ class AutowirePass extends AbstractRecursivePass
     private $ambiguousServiceTypes = array();
     private $autowired = array();
     private $lastFailure;
+    private $throwOnAutowiringException;
+    private $autowiringExceptions = array();
```

Comment:
Good catch! I'm going to clear `$autowiringExceptions` just like the others are cleared.